renderer/components/Bookings.tsx

```diff
@@ -0,0 +1,254 @@
+import React, { useState, useEffect, useMemo } from "react";
+import { useMainStore } from "../store/mainStore";
+import { shallow } from "zustand/shallow";
+import { getStringDate } from "../utils/datetime-ui";
+import { ReportActivity, formatDuration, parseReport } from "../utils/reports";
+import { ParsedReport, TTUserInfo } from "./Calendar/Calendar";
+import Loader from "./ui/Loader";
+
+const MONTHS = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+] as const;
+
```

Comment:
we already have such variable in Calendar. 
Move it to datatime-ui.ts file and use it in 2 places